openwisp_monitoring/monitoring/configuration.py

```diff
@@ -283,6 +283,74 @@ def _get_access_tech():
             }
         },
     },
+    'max_wifi_clients': {
+        'label': _('Max WiFi Clients'),
+        'name': '{name}',
+        'key': 'max_wifi_clients',
+        'field_name': 'clients',
+        'alert_settings': {'operator': '>', 'threshold': 40, 'tolerance': 60},
+        'notification': {
+            'problem': {
+                'verbose_name': 'Max WiFi clients PROBLEM',
+                'verb': _('has more than'),
+                'level': 'warning',
+                'email_subject': _(
+                    '[{site.name}] PROBLEM: Max WiFi Clients exceeded on {notification.target}'
+                ),
+                'message': _(
+                    'The device [{notification.target}]({notification.target_link}) '
+                    '{notification.verb} {notification.actor.alertsettings.threshold} '
+                    'WiFi clients connected.'
+                ),
+            },
+            'recovery': {
+                'verbose_name': 'Max WiFi clients RECOVERY',
+                'verb': _('has returned to normal levels'),
+                'level': 'info',
+                'email_subject': _(
+                    '[{site.name}] RECOVERY: {notification.target} WiFi clients {notification.verb}'
+                ),
+                'message': (
+                    'The device [{notification.target}]({notification.target_link}) '
+                    'WiFi clients {notification.verb}.'
+                ),
+            },
+        },
+    },
+    'min_wifi_clients': {
+        'label': _('Min WiFi Clients'),
+        'name': '{name}',
+        'key': 'min_wifi_clients',
+        'field_name': 'clients',
+        'alert_settings': {'operator': '<', 'threshold': 0, 'tolerance': 240},
```

Comment:
```suggestion
        'alert_settings': {'operator': '<', 'threshold': 1, 'tolerance': 1440},
```

Shouldn't this be less than 1? Would less than zero trigger at all?

I would set a tolerance of 3 days by default (4320 minutes), which would indicate an AP hasn't been used for 3 days in a row, which would account for a full weekend plus festive day, a common occurrence in many countries.